pkg/controller/user.go

```diff
@@ -18,18 +18,22 @@ import (
 	"context"
 
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/logging"
 )
 
 // contextKey is a unique type to avoid clashing with other packages that use
 // context's to pass data.
-type contextKey struct{}
+type contextKey string
```

Comment:
The context docs explicitly say to use something other than a string type. I think these need to be `type struct` instead of `var struct` - I'll clean this up later.